devspace.yaml

```diff
@@ -5,197 +5,31 @@ version: v2beta1
 dependencies:
   ark-cluster-memory:
     path: ./services/ark-cluster-memory
-    pipeline: dev
+  ark-api:
+    path: ./services/ark-api
+  ark-dashboard:
+    path: ./services/ark-dashboard
+  ark:
+    path: ./ark
 #   ark-mcp:
 #     path: ./services/ark-mcp
 #     pipeline: dev
 #   postgres-memory:
 #     path: ./services/postgres-memory
 
-# Hooks for pre-build setup
-hooks:
-  - command: ./services/ark-api/ark-api/scripts/ensure-ark-sdk-wheel.sh
-    events: ["before:dev", "before:deploy"]
-
-# Deployment pipelines define the workflow for different commands
 pipelines:
-  # 'devspace deploy' - Build and deploy production images
   deploy: |-
-    # Deploy any dependencies.
-    run_dependency_pipelines --all --set-flag pipeline=deploy
-    # Build the ark controller and ark service images.
+    run_dependency_pipelines --all --pipeline=deploy
```

Comment:
ahh nice i wondered why it wasn't working for me, knew i was doing something silly